Cargo.toml

```diff
@@ -18,20 +18,20 @@ yaml = ["serde_yaml"]
 parse-value = ["pear"]
 test = ["tempfile", "parking_lot"]
 # toml = ["toml"]
-#Introduces the atomic crate to guarantee compatability with older processors
-fallback-atomic = ["atomic"]
 
 [dependencies]
 serde = { version = "1.0" }
 uncased = "0.9.3"
-atomic = { version = "0.5.0", optional = true }
 pear = { version = "0.2", optional = true }
 toml = { version = "0.5", optional = true }
 serde_json = { version = "1.0", optional = true }
 serde_yaml = { version = "0.8", optional = true }
 tempfile = { version = "3", optional = true }
 parking_lot = { version = "0.11", optional = true }
 
+[target.'cfg(not(target_pointer_width = "64"))'.dependencies]
```

Comment:
This particular incantation means that we enable the dependency even on architectures with > 64 bit wide points. We probably want `any(width = 8, width = 16, width = 32)`.